legiti/src/test/java/com/legiti/UnitTests.kt

```diff
@@ -1,19 +1,22 @@
 package com.legiti
 
 import android.content.Context
+import android.os.Build
 import org.junit.Test
 import org.junit.Assert.*
 import org.junit.runner.RunWith
 import org.robolectric.RobolectricTestRunner
 import org.mockito.*
 import com.legiti.helpers.InvalidCredentials
+import org.robolectric.annotation.Config
 import java.util.*
 
 
 @RunWith(RobolectricTestRunner::class)
+@Config(sdk = intArrayOf(Build.VERSION_CODES.O_MR1))  // So it runs using java 9
 class UnitTests {
 
-    private val AUTH_TOKEN = "sandbox_eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwcmluY2lwYWxJZCI6Imluc3BldG9yX3Rlc3QifQ.cJimBzTsFCC5LMurLelIax_-0ejXYEOZdYIL7Q3GEEQ"
+    private val AUTH_TOKEN = "internal_sandbox_eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwcmluY2lwYWxJZCI6Imluc3BldG9yX3Rlc3QifQ.cJimBzTsFCC5LMurLelIax_-0ejXYEOZdYIL7Q3GEEQ"
```

Comment:
hm is that a valid auth token? (`internal_sandbox_....` )

Reply:
Nope

Reply:
but like why do we want to use this one that's not even a recognize log token format?

Reply:
ah nvm i just saw this comment https://github.com/legiti/legiti-android/pull/33#discussion_r463221124